src/EnergyPlus/Furnaces.cc

```diff
@@ -601,10 +597,6 @@ namespace Furnaces {
                                                                 thisFurnace.HeatingCoilSensDemand,
                                                                 Dummy,
                                                                 thisFurnace.OpMode,
-                                                                thisFurnace.WSHPRuntimeFrac,
-                                                                thisFurnace.MaxONOFFCyclesperHour,
-                                                                thisFurnace.HPTimeConstant,
-                                                                thisFurnace.FanDelayTime,
```

Comment:
Ditto...there are a lot of places where we don't have to pass these in any more. Reviewers should be able to piece it together in similar cases.